include/deal.II/base/symmetric_tensor.h

```diff
@@ -593,10 +593,12 @@ class SymmetricTensor
    * is symmetric only up to round-off, then you may want to call the
    * <tt>symmetrize</tt> function first. If you aren't sure, it is good
    * practice to check before calling <tt>symmetrize</tt>.
+   *
+   * If you want to use this function in a `constexpr` context,
+   * use symmetrize() instead.
```

Comment:
Would you mind elaborating why this is so? I suspect saying that "Because conversion constructors are not allowed in `constexpr` expressions, you will have to use the `symmetrize()` function in such contexts." would be enough. (I'm assuming that this is the reason.)